templates/public/views/find_your_legislator.html

```diff
@@ -6,16 +6,21 @@
 {% block og_title %}Find Your State Legislators - Open States{% endblock %}
 {% block description %}Find out who represents you in your state legislature with Open States.{% endblock %}
 {% block scripts %}
-  <script src="{% static "bundles/fyl.js" %}"></script>
+    <script src="{% static "bundles/fyl.js" %}"></script>
 {% endblock %}
 
 {% block content %}
-<h1 class="heading--large">Find Your Legislators</h1>
-<h2 class="mb2 heading--medium">Look your legislators up by address or use your current location.</h2>
+    <h1 class="heading--large">Find Your Legislators</h1>
+    <h2 class="mb2 heading--medium">Look your legislators up by address or use your current location.</h2>
+    <div>Open States reflects districts for people currently in office for 2022. Legislators and district boundaries
```

Comment:
Might be worth clarifying "current year" vs. "2022/2023"? That way in 10 years we don't have to do this again?

Reply:
Would rather have us be specific and have to update/pay attention to the site a bit more.